src/libraries/System.Memory/ref/System.Memory.cs

```diff
@@ -732,4 +732,37 @@ namespace System.Text
         public System.Text.SpanRuneEnumerator GetEnumerator() { throw null; }
         public bool MoveNext() { throw null; }
     }
+    public readonly partial record struct RunePosition(System.Text.Rune Rune, int StartIndex, int Length, bool WasReplaced)
+        : System.IEquatable<System.Text.RunePosition>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
```

Comment:
AFAIK `private` members don't need to get into _ref-assemblies_ (not like header files in C/C++).
I see other private members in this file that already exists, but is this needed / should be cleaned up?

Reply:
The object field is needed to make the compiler see that the struct is managed, but this one is not, so we don't have to add it.

`_dummyPrimitive` is needed because if the struct had no fields, [the compiler would allow declaring uninitialized variables to it](https://learn.microsoft.com/en-us/dotnet/core/compatibility/2.1#private-fields-added-to-built-in-struct-types).

```suggestion
        private readonly int _dummyPrimitive;
```

Reply:
Thanks!